service-itest-runner

```diff
@@ -11,6 +11,11 @@ IMAGE_NAME="clusterman-dev-$(whoami)"
 docker inspect --type=image "${IMAGE_NAME}" > /dev/null || \
     IMAGE_NAME="docker-paasta.yelpcorp.com:443/services-clusterman:paasta-$(git rev-parse HEAD)"
 DISTRIB_CODENAME=$(docker run -t "${IMAGE_NAME}" lsb_release -cs | tr -d '\n\r')
+if [ "${DISTRIB_CODENAME}" = "xenial" ]; then
+    CIDR_BLOCK="10.0.0.0/24"
+else
+    CIDR_BLOCK="11.0.0.0/24"
+fi
```

Comment:
I know this is just ported from run_instance.py, but what's the deal with this? why do we use different IP ranges for different OSes?

Reply:
When they run in parallel on Jenkins, they'll often end up on the same box, and then they need different IP ranges.  I guess I can leave a comment here.